### PR TITLE
r 3.2.3

### PR DESCRIPTION
**Pull request by amoeba**
Updates the R formula for version 3.2.3. See http://stat.ethz.ch/R-manual/R-devel/doc/html/NEWS.html. Tested on a Mac OS 10.11 system. R installs and the ggplot2 package installs.

### CODE REVIEW

File: r.rb [hunk 1]
```diff
@@ -7,17 +7,9 @@ def stage
 class R < Formula
   desc "Software environment for statistical computing"
   homepage "http://www.r-project.org/"
-  url "http://cran.rstudio.com/src/base/R-3/R-3.2.2.tar.gz"
-  mirror "http://cran.r-project.org/src/base/R-3/R-3.2.2.tar.gz"
-  sha256 "9c9152e74134b68b0f3a1c7083764adc1cb56fd8336bec003fd0ca550cd2461d"
-  revision 1
-
-  bottle do
```
Comment (dunn) [inline r.rb]: Leave the bottle block, our CI bot handles is.

**Comment by amoeba** [comment]
I added a new commit putting back in the bottle section but that's not a very clean fix. Want me to make a new PR?

**Comment by amoeba** [comment]
I reset and made a clean, one-commit update commit.

**Comment by amoeba** [comment]
I'm seeing the CI errors and am wondering if there's anything I could do. I'd be happy to make any changes to the formula.

**Comment by tdsmith** [comment]
The failures are spurious; this looks good. poretools needs to be rebottled, though, I guess.

Some comments on the release notes:

> configure uses pkg-config for PNG, TIFF and JPEG where this is available. This should work better with multiple installs and with those using static libraries.

R builds fine without pkg-config using superenv so it's probably not worth including a pkg-config dependency now but we can keep an eye on this.

> The underlying changes mean that packages providing graphics devices (e.g., Cairo, RSvgDevice, cairoDevice, tikzDevice) will need to be reinstalled.

This is annoying and will break things but there's not much we can do about it.

Thanks very much for the PR!

**Comment by tdsmith** [comment]
@BrewTestBot test this please

Jenkins didn't actually build this for whatever reason.

**Comment by amoeba** [comment]
@tdsmith Thanks, and thanks for the helpful review of the changelog. I'll keep an eye on this for next time.

**Comment by dpo** [comment]
@BrewTestBot test this please

**Comment by dpo** [comment]
I wonder why `littler` fails:
```
==> /usr/local/Cellar/littler/0.2.3_1/bin/littler -e 'print(pi)'
cannot find system Renviron
Fatal error: unable to open the base package
```

**Comment by tdsmith** [comment]
And only on Mavericks and Yosemite, on both Travis and Jenkins? :/

**Comment by tdsmith** [comment]
Looks like revision-bumping worked. Thanks for everyone's help!

**Comment by amoeba** [comment]
Thanks @tdsmith!

**Pull request closed without merging**